src/runtime/content.cpp

```diff
@@ -350,6 +374,72 @@ bool TrContentRuntime::recvXRCommand(int timeout)
   }
 }
 
+bool TrContentRuntime::recvInspectorCommand(int timeout)
+{
+  if (TR_UNLIKELY(inspectorCommandChanReceiver == nullptr))
+    return false;
+
+  inspector_comm::TrInspectorCommandMessage message;
+  if (inspectorCommandChanReceiver->recvCommand(message, timeout))
+  {
+    // Handle incoming inspector command message
+    handleInspectorCommandMessage(message);
+    return true;
+  }
+  else
+  {
+    return false;
+  }
+}
+
+void TrContentRuntime::handleInspectorCommandMessage(inspector_comm::TrInspectorCommandMessage &message)
+{
+  switch (message.getType())
+  {
+  case inspector_comm::TrInspectorCommandType::CdpResponse:
+  {
+    auto cdpResponse = inspector_comm::TrInspectorCommandBase::CreateFromMessage<inspector_comm::TrCdpResponse>(message);
+    DEBUG(LOG_TAG_INSPECTOR, "Content %d: Received CDP response for request %u", id, cdpResponse.requestId);
+
+    // Forward response back to the ContentDomainProxy via the inspector CDP handler
+    auto cdpHandler = getInspectorCdpHandler();
+    if (cdpHandler)
+    {
+      auto contentProxy = cdpHandler->getContentProxy();
+      if (contentProxy)
+      {
+        contentProxy->handleResponseFromContent(cdpResponse.requestId, cdpResponse.cdpResponseJson);
```

Comment:
handleResponseFromContent() is a private method of contentProxy, please fix the issue.